.evergreen/ocsp/mock_ocsp_revoked.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+# Use the CA as the OCSP responder (see https://community.letsencrypt.org/t/unable-to-verify-ocsp-response/7264)
+python3 ocsp_mock.py \
+  --ca_file ca_ocsp.pem \
+  --ocsp_responder_cert ca_ocsp.crt \
+  --ocsp_responder_key ca_ocsp.key \
+   -p 8100 \
```

Comment:
Where does port 8100 come from? Curious if this would conflict with anything else drivers might run and whether it may need to be configurable. I suppose we could have optional shell script arguments and default to whatever value we like.

http://wiki.cacert.org/OcspResponder suggests 2560 (seemingly reference to the RFC).

Reply:
The port is pseudo-configurable via the -p parameter. If we updated the port, we'd need to update the certs. (see https://github.com/mongodb/mongo/blob/master/jstests/ssl/x509/certs.yml#L359 for where we configure this).

 Ah, interesting find on the suggested port! As a counterpoint though, Let's Encrypt appears to be using port 80 (OCSP - URI:http://isrg.trustid.ocsp.identrust.com, as show by the output of `openssl s_client -host valid-isrgrootx1.letsencrypt.org -port 443 -prexit -showcerts`)

I'd recommend leaving the port the same for now as it's simpler, although if a conflict does exist, we can change it.

Reply:
Fine keeping 8100 and reconsidering if/when a conflict comes up. I didn't realize that it was hard-coded into the certs, but I suppose that makes sense now that I think about it.